doc/userguide/src/ExecutingTestCases/ConfiguringExecution.rst

```diff
@@ -165,6 +165,25 @@ running tests.
 
 .. note:: :option:`--ProcessEmptySuite` option was added in Robot Framework 2.7.2.
 
+Re-executing failed test suites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Command line option :option:`rerunfailedsuites (-S)` can be used to select all failed
+suites from an earlier `output file`_ for re-execution. Like  :option:`--rerunfailed (-R)`, this option is useful when
+full test execution takes a lot of time. Note that all tests from a failed test suite will be re-executed, even
+passing ones. This option is useful when the tests of a test suite depends on each other.
+
+::
+
+  robot tests                                  # first execute all tests
+  robot --rerunfailedsuites output.xml tests    # then re-execute failing
```

Comment:
Do we really need an example here when there's one with `--rerunfailed` and these features work exactly same way?